### PR TITLE
Filter credit transaction table by fuel supplier

### PR DESCRIPTION
**Pull request by plasticviking**
Addresses story https://trello.com/c/RqkFy28i

### CODE REVIEW

File: frontend/src/credit_transfers/components/CreditTransactionsPage.js [hunk 1]
```diff
@@ -8,20 +8,42 @@ import { DEFAULT_ORGANIZATION } from '../../constants/values';
 import history from '../../app/History';
 import Loading from '../../app/components/Loading';
 import CreditTransferTable from './CreditTransferTable';
+import {organization} from "../../sampleData";
```
Comment (amichard) [inline frontend/src/credit_transfers/components/CreditTransactionsPage.js]: Is this a sample test code?

Comment (plasticviking) [reply frontend/src/credit_transfers/components/CreditTransactionsPage.js]: It's an unused import. I'll clean it up.

**Comment by amichard** [comment]
Can you also look into fixing the eslint issues?
screen shot 2018-05-31 at 8 32 55 am

**Comment by plasticviking** [comment]
@amichard Done. I had been accidentally been using my global version of .eslintrc rather than the project one.

**Pull request merged**